src/modules/welcome-page/index.js

```diff
@@ -10,15 +11,13 @@ class WelcomePage extends Component {
     return (
       <div className='welcome-page-container'>
         <Header />
-        <BrowserRouter>
-          <Switch>
-            <Route path='/registration' component={Registration} />
-            <Route path='/login' component={Login} />
-          </Switch>
-        </BrowserRouter>
+        <Switch>
```

Comment:
We do not need switch in this case. There is no possibility that two routes will match.